_includes/templates/forms/plain-writing-feedback-2-form.html

```diff
@@ -0,0 +1,294 @@
+<div class="plain-writing-feedback">
+    <section class="block">
+        <div class="content-l
```

Comment:
Should this be an ordered list with custom list-items, like the submit a request form `/initiatives/submit-a-request/`? That one had placeholder images in the wireframe as well.